tools/ARIAtools/extractProduct.py

```diff
@@ -387,20 +378,24 @@ def prep_dem(demfilename, bbox_file, prods_TOTbbox, prods_TOTbbox_metadatalyr,
     return aria_dem, ds_aria, Latitude, Longitude
 
 def dl_dem(path_dem, path_prod_union, num_threads):
-    """Download the DEM over product bbox union."""
-
+    """
+        Download the DEM over product bbox union.
+    """
     # Import functions
     from ARIAtools.shapefile_util import shapefile_area
 
     root      = os.path.splitext(path_dem)[0]
     prod_shapefile = open_shapefile(path_prod_union, 0, 0)
     WSEN      = prod_shapefile.bounds
-    # If area > 225000 km2, must split requests into chunks to successfully access data
+    # If area > 450000 km2, must split requests into chunks to successfully access data
     chunk = False
-    if shapefile_area(prod_shapefile) > 300000:
+    if shapefile_area(prod_shapefile, bounds = True) > 450000:
```

Comment:
@sssangha suggest to be conservative and make this sligthly smaller than the openTopo issue.

Reply:
```suggestion
    if shapefile_area(prod_shapefile, bounds = True) > 400000:
```